internal/content_coding.go

```diff
@@ -264,6 +283,41 @@ func (e *ZlibEncoder) Encode(data []byte) ([]byte, error) {
 	return e.buf.Bytes(), nil
 }
 
+type ZstdEncoder struct {
+	encoder *zstd.Encoder
+}
+
+func NewZstdEncoder(options ...EncodingOption) (*ZstdEncoder, error) {
+	cfg := encoderConfig{level: -1}
+	for _, o := range options {
+		o(&cfg)
+	}
+
+	// Map the levels
+	var level zstd.EncoderLevel
+	switch cfg.level {
+	case 1:
+		level = zstd.SpeedFastest
+	case -1, 3:
```

Comment:
Why -1 is with 3??

Reply:
Because I set `-1` in as "default" some lines above.